pages/index.tsx

```diff
@@ -0,0 +1,13 @@
+import Layout from '../components/Layout';
+import { useFetch } from '../lib/hooks';
+import { HakkasanResponse } from '../lib/hakkasan';
```

Comment:
Nit: It might be a good idea to prefix TypeScript type imports using `type` so that it's clear that what's being imported is a typing and not an object literal:
```ts
import type { HakkasanResponse } from '../lib/hakkasan';
```

Reply:
Hehe yes I remembered you like this - forgot to do it here though! Good catch!

Reply:
Yeees much better 😌 Resolving!